test/synapse/2019-06-01/azartifacts/zz_generated_library.go

```diff
@@ -50,8 +50,8 @@ func (client *libraryClient) appendCreateRequest(ctx context.Context, libraryNam
 	query := req.URL.Query()
 	query.Set("api-version", "2019-06-01-preview")
 	req.URL.RawQuery = query.Encode()
-	if options != nil && options.XMSBlobConditionAppendpos != nil {
-		req.Header.Set("x-ms-blob-condition-appendpos", strconv.FormatInt(*options.XMSBlobConditionAppendpos, 10))
+	if options != nil && options.BlobConditionAppendpos != nil {
```

Comment:
XMS was removed here

Reply:
This is expected.  However, there is inconsistency on where "XMS" is removed.  Historically it was removed everywhere.  Presumably we want to maintain the same pattern.